src/s1reader/s1_orbit.py

```diff
@@ -1,12 +1,169 @@
+import cgi
 import datetime
 import glob
 import os
+import requests
 import warnings
 
-
 # date format used in file names
 FMT = "%Y%m%dT%H%M%S"
 
+# Required for orbit download
+scihub_url = 'https://scihub.copernicus.eu/gnss/odata/v1/Products'
+# Namespaces of the XML file returned by the S1 query. Will they change it?
+w3_url = '{http://www.w3.org/2005/Atom}'
+m_url = '{http://schemas.microsoft.com/ado/2007/08/dataservices/metadata}'
+d_url = '{http://schemas.microsoft.com/ado/2007/08/dataservices}'
```

Comment:
Since `scihub_url`, `m_url`, and `d_url` are only used in [`get_orbit_dict`](https://github.com/opera-adt/s1-reader/blob/970ea7ed2d4aa56fe642d63010a7b7db09efb63c/src/s1reader/s1_orbit.py#L64), let's move them there; Unless there's a reason for them to be global and importable elsewhere.